client/state/data-layer/analytics/adTracking.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { recordSelectedVertical } from 'lib/analytics/ad-tracking';
+import { SIGNUP_STEPS_SURVEY_SET } from 'state/action-types';
+
+function handleSignupStepsSurveyStart( _, action, next ) {
```

Comment:
note also that we discourage the naming of `handle` prefixes. in any respect, this function doesn't _handle_ so much as it _tracks_ or _records_.

something like `recordVerticalSelection` might make more sense as a name here